tools/vpdbootmanager/get_test.go

```diff
@@ -5,49 +5,13 @@
 package main
 
 import (
-	"bytes"
 	"testing"
-
-	"github.com/u-root/u-root/pkg/vpd"
 )
 
-var getter *Getter
-
-func TestNewgetter(t *testing.T) {
+func TestNewGetter(t *testing.T) {
 	if g := NewGetter(); g == nil {
 		t.Fatalf(`NewGetter() = %v, want not nil`, g)
 	} else if g.R == nil || g.Out == nil {
 		t.Errorf(`g.R, g.Out = %v, %v, want not nil, not nil`, g.R, g.Out)
 	}
 }
-
-// testGetOne is failing
-func testGetOne(t *testing.T) {
```

Comment:
let's fix this test, not remove it.

Reply:
The easy way to keep it top of mind: rename to TestGetOne
add t.Skip to it with a message such as "skipping, please fix" or some such.

Reply:
sgtm